src/app.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const fs = require('fs');
+
+function funcRead() {
+  const terminal = process.argv.slice(2);
+  const filePath = './src/' + terminal[0];
+  const newFilePath = './src/' + terminal[1];
```

Comment:
You should use template literals instead of concatenating strings with '+'. This improves readability and is the recommended way to handle string interpolation in modern JavaScript.